### ISSUE TITLE
10 Keras test failures, all complaining of the same 4 missing symbols.

### ISSUE THREAD

**Issue by capn-freako** [plaidml/plaidml#1364]
Hi all,

After successfully building `plaidml-v1:HEAD (ecf94987...)` on a MacBook, I'm getting 10 _Keras_ test failures, all with the same last line in their Python backtraces:

```
plaidml.ffi.Error: Failed to materialize symbols: { (main, { __mlir_get_value, __mlir_ciface_get_value, _get_value }) }
```

Is this a known issue, re: missing packages, libraries, etc.?

Thanks,
-db

Commands executed:

1. `bazelisk build //plaidml:wheel`
2. `pip install -U bazel-bin/plaidml/wheel.tmp/tmp/dist/*.whl`
3. `bazelisk run //plaidml:setup` (`llvm_cpu0` selected)
4. `bazelisk build //plaidml/bridge/keras:wheel`
5. `pip install -U bazel-bin/plaidml/bridge/keras/wheel.tmp/tmp/dist/plaidml_keras-0.0.0.dev0-py2.py3-none-any.whl`
6. `bazelisk run //plaidml/bridge/keras:backend_test`

**Comment by flaub** [comment]
Thanks for the report, I'll look into this.

**Comment by flaub** [comment]
BTW, you can easily check the repro with:
```
bazelisk run //plaidbench:bin -- -n1 keras resnet50
```

Or you can run the backend test directly with:
```
bazelisk run //plaidml/bridge/keras:backend_test
```

**Comment by flaub** [comment]
Should be fixed in #1368
